lib/iiif_print/page_derivative_service.rb

```diff
@@ -22,7 +22,7 @@ def valid?
       # fallback to Fedora-stored relationships if work's aggregation of
       #   file set is not indexed in Solr
       parent = file_set.member_of.find(&:work?) if parent.nil?
-      IiifPrint.config.work_types_for_derivative_service.include?(parent.class)
+      parent.iiif_print_config?
```

Comment:
Doesn't it need to be: try(:iiif_print_config?) because as far as I know, we don't yet have a way to handle this method if iiif_print isn't included in the model.

Reply:
Good point!